lib/i18n/activities.js

```diff
@@ -49,3 +49,38 @@ export const ActivityLabelI18n = defineMessages({
     defaultMessage: 'Ticket confirmed',
   },
 });
+
+export const ActivityCategoryLabelI18n = defineMessages({
+  ALL: {
+    id: 'AllActivity',
+    defaultMessage: 'All activities',
+  },
+  COLLECTIVE: {
+    id: 'CollectiveActivity',
+    defaultMessage: 'Collective activity',
+  },
+  EXPENSES: {
+    id: 'ExpensesActivity',
+    defaultMessage: 'Expenses activity',
+  },
+  CONTRIBUTIONS: {
+    id: 'ContributionsActivity',
+    defaultMessage: 'Contributions activity',
+  },
+  ACTIVITIES_UPDATES: {
+    id: 'UpdatesActivity',
+    defaultMessage: 'Updates activity',
+  },
+  VIRTUAL_CARDS: {
+    id: 'VirtualCardActivity',
+    defaultMessage: 'Virtual Card activity',
+  },
+  FUND_EVENTS: {
+    id: 'FundEventActivity',
+    defaultMessage: 'Fund Event activity',
+  },
+  REPORTS: {
+    id: 'ReportsActivity',
+    defaultMessage: 'Reports activity',
+  },
```

Comment:
Sounds like all these should be plural (e.g. "Expenses activities"). Worth sharing a screenshot on Slack's #wording channel to ask for some feedback.

Reply:
Changed everything to "... activities" and created a thread on the wording channel; https://opencollective.slack.com/archives/C019YCG6RCY/p1660831885324169